app/views/shared/_navbar.html.erb

```diff
@@ -10,11 +10,8 @@
 
   <div class="collapse navbar-collapse" id="navbarSupportedContent">
     <ul class="navbar-nav mr-auto">
-      <li class="nav-item active">
-        <%= link_to "Home", "#", class: "nav-link" %>
-      </li>
-      <li class="nav-item">
-        <%= link_to "Messages", "#", class: "nav-link" %>
+      <li class="nav-item" id="nav-btn">
+        <%= link_to "login", "#", class: "nav-link" %>
```

Comment:
Would change to 

`<% if current_user %>`
    <%= link_to "login", "#", class: "nav-link" %>
<% end %>